File: 01.JS-Basic/FOR-ЦИКЪЛ УПРАЖНЕНИЕ/Salary.js
function salary(input) {

    let openTabs = Number(input[0])
    let salary = Number(input[1])
    let penalty = 0

    for (let i = 2; i <= input.length - 1; i++) {

        let tab = input[i]

        switch (tab) {
            case "Facebook": penalty += 150; break;
            case "Instagram": penalty += 100; break;
            case "Reddit": penalty += 50; break;
        }
    }

    if (salary - penalty <= 0) {
        console.log("You have lost your salary.")
    }
    else {
        console.log(salary - penalty)
    }
}

salary(["3", "500", "Facebook", "Stackoverflow.com", "softuni.bg"])

File: JAVA-Basic/JavaBasic/src/ForLoop/Exercise/Salary.java
package ForLoop.Exercise;

import java.util.Scanner;

public class Salary {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int openTabs = Integer.parseInt(scan.nextLine());
        int salary = Integer.parseInt(scan.nextLine());

        for (int i = 0; i < openTabs; i++) {
            String checkTab = scan.nextLine();

            switch (checkTab) {
                case "Facebook":
                    salary -= 150;
                    break;
                case "Instagram":
                    salary -= 100;
                    break;
                case "Reddit":
                    salary -= 50;
                    break;
            }
        }
        if (salary <= 0) {
            System.out.println("You have lost your salary.");
        }else {
            System.out.println(salary);
        }
    }
}
